src/test/scala/uk/gov/hmrc/test/ui/pages/base/PageHelper.scala

```diff
@@ -21,7 +21,7 @@ import scala.util.matching.Regex
 trait PageHelper extends DriverHelper {
 
   private val initPart: String = "/declaration"
-  private val elementId: String = "[0-9A-Za-z\\-]+"
+  private val elementId: String = "[0-9A-Za-z\\-.]+"
```

Comment:
The dot must must be prefixed with `\\` because the regex will not work otherwise.
A dot in a regex means "any character, I do not care which one"

Reply:
done